task/__init__.py

```diff
@@ -400,10 +400,11 @@ def label(issue, labels=('bot',)):
     return api.post(resource, labels)
 
 
-def labels_of_pull(pull):
-    if "labels" not in pull:
-        pull["labels"] = api.get(f"issues/{pull['number']}/labels")
-    return [label["name"] for label in pull["labels"]]
+def labels_of_pull(pull: JsonObject) -> Sequence[str]:
+    if "labels" in pull:
+        return [get_str(label, "name") for label in get_dictv(pull, "labels")]
+    else:
+        return api.get(f"issues/{pull['number']}/labels")
```

Comment:
This looks suspicious.  I think this is going to end up returning a sequence of JSON objects instead of a sequence of strings.

Consider that the old code always did the unpacking step, also on the result of `.get()`.

The reason this didn't raise alarm bells is because `.get()` returns `Any` (ie: is unsafe).  Please try to use `.get_objv()` when fixing this.

Reply:
Erk, yes, what nonsense.. Fixed, thanks!